src/internal/components/drawer/interfaces.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BaseComponentProps } from '../../base-component';
+import React from 'react';
+
+export interface DrawerProps extends BaseComponentProps {
+  /**
+   * Header of the drawer.
```

Comment:
Have these doc strings ben reviewed by content design?

Reply:
They are copied word for word from the Help Panel documentation, other than `drawer` instead of `panel`. So they were reviewed when the help panel was released.